src/main/java/org/carlmontrobotics/robotcode2023/commands/IntakeRoller.java

```diff
@@ -0,0 +1,56 @@
+package org.carlmontrobotics.robotcode2023.commands;
+
+import java.awt.Color;
+
+import org.carlmontrobotics.robotcode2023.subsystems.Roller;
+import org.carlmontrobotics.robotcode2023.subsystems.Roller.RollerMode;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class IntakeRoller extends CommandBase {
+
+    // .5 for cones intake
```

Comment:
omit these comments (Some of the more trivial suggestions I'm commenting on I can fix them immediately, this is just to remind me once I pull up vscode to actually do it)